bindings/encoding/struct.go

```diff
@@ -35,6 +36,10 @@ type TaikoL1Evidence struct {
 
 // FromGethHeader converts geth *types.Header to *BlockHeader
 func FromGethHeader(header *types.Header) *BlockHeader {
+	baseFeePerGas := header.BaseFee
+	if baseFeePerGas == nil {
+		baseFeePerGas = new(big.Int).SetInt64(0)
+	}
```

Comment:
No need these code, because `BaseFeePrGas *big.Int` with default value `nil` in go, so BlockHeader.BaseFeePerGas will be `nil` or some value, the behavior is same with `header *types.Header`